style.css

```diff
@@ -0,0 +1,397 @@
+* {
+	box-sizing:border-box;
+	scroll-behavior: smooth;
+}
+
+.nav-link {
+	display: flex;
+	justify-content: flex-start;
+	gap: 20px;
+	align-items: center;
+}
+
+img[alt="french"] {
+	max-width:30px;
+	border:1px solid white;
+	border-radius: 15px;
+}
+
+img[alt="UK"] {
+	max-width:30px;
+	border:1px solid white;
+	border-radius: 15px;
+}
+.nav-link a {
+	text-decoration: none;
+	color:#FAFAFA;
+	font-weight: bold;
+}
+
+.nav-link a:hover {
+	color: #FF7E67;
+	display: flex;
+	height: 2rem;
```

Comment:
surely font size also determines the height so you don't have to set the height?